client/src/components/modals_and_popovers/FixedPopover.stories.tsx

```diff
@@ -9,9 +10,17 @@ export default {
 
   // Need decorators to use useArgs()
   decorators: [(Story) => <div>{Story()}</div>],
-};
+} as Meta;
+
+type ComponentProps<T> = T extends
+  | React.ComponentType<infer P>
+  | React.Component<infer P>
+  ? JSX.LibraryManagedAttributes<T, P>
+  : never;
```

Comment:
Just a note, #1191 is probably going to declare this utility type centrally in `src/types/utility_Types.d.ts`. Depending which branch merges first, please coordinate with Ivan to switch to using the central definition here.

Reply:
I'm going to wait for Ivan's branch to merge first then I'll rebase off of master and adjust mine since he added the utility type centrally.